corehq/apps/app_manager/static/app_manager/js/forms/advanced/actions.js

```diff
@@ -576,7 +576,7 @@ hqDefine('app_manager/js/forms/advanced/actions', function () {
 
             self.propertyCounts = ko.computed(actionBase.propertyCounts(self));
 
-            self.name_path = ko.computed(function () {
+            self.name_update = ko.computed(function () {
```

Comment:
Are you certain this needs to change? Calling this `name_update` makes me think that it's dealing with a ConditionalCaseUpdate, but it's only dealing with a string.